cmd/entrypoint/subcommands/decode_script.go

```diff
@@ -0,0 +1,56 @@
+package subcommands
+
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+	"io"
+	"io/fs"
```

Comment:
small note: this forces us to us Go 1.16 to build tekton 📦 
cc @nikhil-thomas @chmouel @piyush-garg

Reply:
I don't think we would be able to upgrade to go1.16 for productized RHEL for quite some time,

Reply:
ahhh ok good to know! i will tackle this without the fs mock.

Reply:
Removed the features that required go 1.16 and have tested with go1.15.13 to confirm everything's passing as expected.